modules/auxiliary/client/telegram/send_message.rb

```diff
@@ -0,0 +1,55 @@
+# Content: Metasploit Module to send telegram message
+# Author: Ege Balcı| @egeblc | https://pentest.blog
+# Date: 07/2020
```

Comment:
Sorry but we require the following header in all of our modules. Please do not remove this header:

```
##
# This module requires Metasploit: https://metasploit.com/download
# Current source: https://github.com/rapid7/metasploit-framework
##
```
You already have the info provided in your comment in the `def initialize` section, where you can add the release date and author, and via the associated documentation file.

Reply:
Fixed and moved the website link and handle to be part of the comment as described. Date will be left to commit history as this is the common way to find what was submitted and when it was submitted.